apps/api/src/schema/websoc.ts

```diff
@@ -75,18 +75,36 @@ const isValidRestrictionCode = (code: string): code is (typeof restrictionCodes)
 export const websocQuerySchema = z.object({
   year: z
     .string({ message: "Parameter 'year' is required " })
-    .length(4, { message: "Parameter 'year' must have length 4" }),
-  quarter: z.enum(terms, { required_error: "Parameter 'quarter' is required" }),
+    .length(4, { message: "Parameter 'year' must have length 4" })
+    .openapi({ description: "The academic year of the course ", example: "2025" }),
+  quarter: z.enum(terms, { required_error: "Parameter 'quarter' is required" }).openapi({
+    description: "The academic quarter of the course",
+    example: "Fall",
+  }),
   ge: z
     .enum(geCategories)
     .optional()
+    .openapi({ description: "The general education category of the course", example: "GE-1A" })
```

Comment:
Comment about singular course and notion of query search parameter vis-a-vis response subfield stands throughout this review.